mm2src/coins/utxo/utxo_builder/utxo_arc_builder.rs

```diff
@@ -104,7 +104,14 @@ where
         self.spawn_merge_utxo_loop_if_required(&utxo_arc, self.constructor.clone());
 
         let result_coin = (self.constructor)(utxo_arc.clone());
-        if let Some(sync_status_loop_handle) = sync_status_loop_handle {
+        let spv_conf = &result_coin.as_ref().conf.spv_conf;
```

Comment:
You can get `spv_conf` using `let spv_conf = utxo.conf.spv_conf.clone();` under this
https://github.com/KomodoPlatform/atomicDEX-API/blob/8b6d3cb0fe3d42f7fa7d0c8b2b501ff8aa33ee0b/mm2src/coins/utxo/utxo_builder/utxo_arc_builder.rs#L101
You won't have to clone it again later :)